config/crd/bases/limitador.kuadrant.io_limitadors.yaml

```diff
@@ -4,7 +4,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.7.0
+    controller-gen.kubebuilder.io/version: v0.4.1
```

Comment:
I *think* you need to delete your `bin` directory and re-generate everything again, this shouldn't be getting changed.

```
$ ./bin/controller-gen --version
Version: v0.7.0
```

Reply:
ok, removed the commit causing the change.